apps/src/lib/kits/maker/commands.js

```diff
@@ -0,0 +1,27 @@
+/** @file Maker commands (invoked by Applab/Gamelab.executeCmd) */
+import {apiValidateType} from '@cdo/apps/javascriptMode';
+
+const commands = {
+
+  /**
+   * Execute some code every X milliseconds.  This is effectively setInterval()
+   * with a cleaner interface.
+   * @param {number} opts.ms How often to invoke the code in the loop,
+   *        in milliseconds.
+   * @param {function(function)} opts.callback Code to invoke in each loop
+   *        iteration. Gets passed an 'exit' function that will stop the loop.
+   */
+  timedLoop(opts) {
```

Comment:
I wonder if we should start specifying these as: `timedLoop({ms, callback}) {` for more clarity. Though I guess apiValidateType needs an object as the first param, so that might not work.